### ISSUE TITLE
PBR Support ?

### ISSUE THREAD

**Issue by rt974** [dertom95/Urho3D-Blender#13]
Hey !

When i pick a PBR Technique, example with no textures, and a PBR params node, the object becomes absolute black !

I have no clue why this fails, from what i have seen in the 42_PBRMaterials sample, there is no particular code to add to the application in order to activate pbr materials, they should work off the shelf ! I've just discovered Urho3D yesterday, and trying to setup a workflow for exporting models and maybe scenes from blender, i find the Urho3D Editor a bit strange and weird, i mean it works, but im way more familiar with Blender for building scenes, it is more convenient.

Let me know if you have any workaround for this, im using the latest Release, not the master, with blender 2.83.

**Comment by dertom95** [comment]
Yeah, you can create the pbr materials but the preview is not switching into pbr mode, so the preview won't work with pbr. To be honest I never used pbr with urho3d and not sure what steps to take....
